images/ubuntu.yaml

```diff
@@ -346,6 +346,7 @@ files:
   - kinetic
   - lunar
```

Comment:
Time to drop kinetic (22.10) and lunar (23.04) as they are both EOL https://fridge.ubuntu.com/2023/12/15/ubuntu-23-04-lunar-lobster-reaches-end-of-life-on-january-25-2024/

Maybe there's a commit doing just that that needs to be cherry-picked?